aspnetcore/security/enforcing-ssl.md

```diff
@@ -411,6 +411,8 @@ dotnet dev-certs https --trust
 
 Close any browser instances open. Open a new browser window to app.
 
+See [this GitHub issue](https://github.com/aspnet/AspNetCore/issues/16892) for troubleshooting certificate issues with Visual Studio.
```

Comment:
Just a nit suggestion ... you know how much I love the "see this" link text. :smile: lol

```suggestion
See [HTTPS Error using IIS Express (aspnet/AspNetCore #16892)](https://github.com/aspnet/AspNetCore/issues/16892) for troubleshooting certificate issues with Visual Studio.
```